lib/survey_gizmo/api/page.rb

```diff
@@ -9,20 +11,27 @@ class Page
     attribute :properties,    Hash
     attribute :after,         Integer
     attribute :survey_id,     Integer
+    attribute :questions,     Array[Question]
```

Comment:
Ok great, that's additional data but it won't break anything. we'll probably try to use it too